<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Group Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Btw. Groups</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Inner Group</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="remove_group_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Remove Selected Group</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_group_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Add a Group</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Global Rule:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rule_result">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_group_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>remove_group()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_group_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>add_group()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>remove_group()</slot>
  <slot>add_group()</slot>
 </slots>
</ui>
